tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py36,py37
+envlist = {py27,py36,py37}-{2.7,3.6,3.7}
```

Comment:
Why this change? Can you please elaborate?

Reply:
In my environment, if i run your command, `tox -e 2.7`, it shows undefined environment, i chose to alias the environment list, so that i can run the tox test, and align with your command written in the issue.

Reply:
You need to run `python -m tox -e py27` instead :) 
You may also run `python -m tox .`

May you instead replace it with this?
`envlist = py2,py3`

Reply:
sure, i will replace with `envlist = py2,py3`